traffic_portal/test/end_to_end/servers/servers-spec.js

```diff
@@ -117,6 +117,20 @@ describe('Traffic Portal Servers Test Suite', function() {
 		});
 	});
 
+	it('should navigate back to the new server and view the delivery services assigned to the server', function() {
+		console.log('Managing the delivery services of ' + mockVals.hostName);
+		browser.navigate().back();
+		pageData.moreBtn.click();
+		pageData.viewDeliveryServicesMenuItem.click();
+		expect(browser.getCurrentUrl().then(commonFunctions.urlPath)).toMatch(commonFunctions.urlPath(browser.baseUrl)+"#!/servers/[0-9]+/delivery-services");
+	});
+
+	it('should ensure you cannot clone delivery service assignments because there are no delivery services assigned to the server', function() {
+		console.log('Ensure you cannot clone delivery service assignments for ' + mockVals.hostName);
+		pageData.moreBtn.click();
+		expect(element(by.css('clone-ds-assignments')).isPresent()).toEqual(false);
```

Comment:
an element that matches the CSS selector `clone-ds-assignments` must be something like:
```html
<clone-ds-assignments>
...
</clone-ds-assignments>
```
but that's not a real HTML element. Is this meant to be a class/id/name?

Reply:
hmm. i guess i did screw that up. no wonder it passed. :)